src/mpyl/cli/repository.py

```diff
@@ -130,11 +130,10 @@ def status(obj: RepoContext):
                 )
             )
         else:
-            changes = repo.changes_in_branch()
             console.log(
                 Markdown(
                     f"Revision for `{repo.main_origin_branch}` not found. Cannot diff with base. "
-                    f"*{len(changes)}* (grafted) commits on `{repo.get_branch}`"
+                    f"*1* (grafted) commits on `{repo.get_branch}`"
```

Comment:
What does "grafted" mean in this context?
Also, if it cannot diff with the base branch, how would it know how many extra commits there are on the current branch? Let's consider removing this message if it's not adding any value.

Reply:
This is part of the `repo status` function that you want to remove anyway, so I won't touch it in this pr.